src/cluster/slot_migrate.cc

```diff
@@ -346,6 +351,7 @@ Status SlotMigrator::sendSnapshotByCmd() {
   auto iter = util::UniqueIterator(storage_->GetDB()->NewIterator(read_options, cf_handle));
 
   // Seek to the beginning of keys start with 'prefix' and iterate all these keys
+  auto current_slot = slot_range.start;
```

Comment:
Seems this assignment can be removed.

Reply:
This also informs the following error handling, considering the case where the first iter fails and does not update the `current_slot`.
```
```
  if (auto s = iter->status(); !s.ok()) {
    auto err_str = s.ToString();
    LOG(ERROR) << "[migrate] Failed to iterate keys of slot " << current_slot << ": " << err_str;
    return {Status::NotOK, fmt::format("failed to iterate keys of slot {}: {}", current_slot, err_str)};
  }
```
```

Reply:
@PokIsemaine Got it, makes sense. Thanks for your explain.